plugin/exo/Services/classes/QTI/QtiServices.php

```
@@ -48,8 +59,34 @@ public function sortPathOfQuestions(QtiRepository $qtiRepo, $stepDir = '')
                 $dirs[] = $pathQtiDir.'/'.$question->getFilename();
             }
         }
+
         sort($dirs);
 
         return $dirs;
     }
+    /**
+     * create the directory questions to export an exercise and export the qti files.
+     *
+     * @param \UJM\ExoBundle\Entity\Question[] $questions
+     * @param int                              $numStep
+     */
+    public function createQuestionsDirectory(array $questions, $numStep)
+    {
+        $qtiRepo = $this->container->get('ujm.exo_qti_repository');
+
+        @mkdir($qtiRepo->getUserDir().'questions');
```

Isn't that better to actually do the check instead of just silence errors ?